src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,15 @@
+import React, {useState} from "react";
+import "../Color/Color.scss";
+
+const ColorButton = (props) => {
+  const {color} = props
+  return (
+    <>
+    <button className={`colorButton ${color}`} onClick={() => {props.setColor(color)} }/>
```

Comment:
```suggestion
    <button className={`colorButton ${color}`} onClick={() => {setColor(color)} }/>
```

위의 리뷰에 따르면 이렇게 변경할 수 있겠네요!

Reply:
수정했습니다~ 확인 부탁드립니다
감사합니다 💕